### DISCUSSION TITLE
how to use toLower() in cypher?

### DISCUSSION THREAD

**Discussion by aizaz-sparkai** [neo4j-contrib/neovis.js#348] [Q&A]
how to use toLower() in cypher?  

my cypher is 

```cypher 
match (n) - [r] - (m)
where toLower(n.name) contains toLower("hello world")
return n, r, m
```

**Comment by thebestnom** [comment]
Hi, this is neovis.js specific repo, Ill try to help, but for next time ask neo4j forum 😅

https://stackoverflow.com/questions/45212213/is-there-any-way-in-neo4j-using-contains-to-compare-case-insensitive-string

I would suggest the second answer as it will more likely will use index if it can

Other than that I think it looks correct, what error do you get?
